layouts/partials/continuous_delivery/cd-pipelines-getting-started.html

```diff
@@ -1,11 +1,39 @@
 {{ $dot := . }}
-<div class="ci-cd-integrations">
-  <div class="container cards-dd col-num-3">
-    <div class="row row-cols-2 row-cols-sm-4 g-2 g-xl-3 justify-content-sm-center">
+<div class="cd-providers">
+  <div class="container cards-dd">
+    <div class="row row-cols-5 g-2 g-xl-3 justify-content-sm-center">
       <div class="col">
         <a class="card h-100" href="/continuous_delivery/deployments/argocd/">
           <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/argocd_small.svg" "class" "img-fluid" "alt" "argocd" "width" "200") }}
+            {{ partial "img.html" (dict "root" $dot "src" "integrations_logos/argocd_small.svg" "class" "img-fluid" "alt" "argocd" "width" "150") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_delivery/deployments/ciproviders/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" $dot "src" "integrations_logos/circleci.png" "class" "img-fluid" "alt" "circleci" "width" "150") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_delivery/deployments/ciproviders/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" $dot "src" "integrations_logos/github_small.svg" "class" "img-fluid" "alt" "github actions" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_delivery/deployments/ciproviders/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" $dot "src" "integrations_logos/gitlab_small.svg" "class" "img-fluid" "alt" "gitlab" "width" "150") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_delivery/deployments/ciproviders/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" $dot "src" "integrations_logos/jenkins.png" "class" "img-fluid" "alt" "jenkins" "width" "150") }}
```

Comment:
Do you think it would be possible to have a general image for CI providers with several logos in it, instead of separated images? I am wondering this because this feature is actually supported for almost all of our providers (also Azure DevOps and Buildkite for example), and the setup for all of them is the same. I would probably have a single image with several providers icon, that points to the `ciproviders` setup if it's feasible